src/rust/bitbox02/src/keystore.rs

```diff
@@ -51,3 +54,26 @@ pub fn create_and_store_seed(password: &Password, host_entropy: &[u8; 32]) -> bo
         bitbox02_sys::keystore_create_and_store_seed(password.as_cstr(), host_entropy.as_ptr())
     }
 }
+
+#[derive(Copy, Clone)]
+struct ZeroizedMnemonic([u8; 256]);
+impl core::default::Default for ZeroizedMnemonic {
+    fn default() -> Self {
+        ZeroizedMnemonic([0; 256])
+    }
+}
+impl zeroize::DefaultIsZeroes for ZeroizedMnemonic {}
+
+pub fn get_bip39_mnemonic() -> Result<zeroize::Zeroizing<String>, ()> {
+    let mut mnemonic = zeroize::Zeroizing::new(ZeroizedMnemonic([0u8; 256]));
```

Comment:
Why not `new(ZeroizedMnemonic(Default::default()))`?

Reply:
how about this:

```rust
let mut mnemonic: zeroize::Zeroizing<ZeroizedMnemonic> = zeroize::Zeroizing::new(Default::default());
```

Reply:
i don't see how that is better?

Reply:
you've already implemented the default anyway. why do it here again. maybe tomorrow you'll want to change the default to `ZeroizedMnemonic([0; 300])`. just feels a bit weird. but anyway, not a big deal.